cmd/image-builder/repos_test.go

```diff
@@ -71,8 +71,8 @@ func TestNewRepoRegistryImplDatadir(t *testing.T) {
 
 	// create a custom datadir with testdistro-1.json, the basefilename
 	// must match a distro nameVer
-	dataDir := t.TempDir()
-	repoFile := filepath.Join(dataDir, "repositories", "testdistro-1.json")
+	repoDir := t.TempDir()
+	repoFile := filepath.Join(repoDir, "repositories", "testdistro-1.json")
```

Comment:
It may be beneficial to test this also without the `repositories` sub-directory.